src/modules/iasql_functions/rpcs/iasql_rollback.ts

```diff
@@ -33,7 +33,9 @@ export class IasqlRollback extends RpcBase {
   /** @internal */
   postTransactionCheck = PostTransactionCheck.UNLOCK_ALWAYS;
   /** @internal */
-  inputTable = {};
+  inputTable = {
+    force: { argType: 'boolean', default: 'false', rawDefault: true },
+  };
```

Comment:
@mtp1376 could you please check if I'm doing the proper use of the RPC `inputTable` and how I'm calling the RPC afterwards?